ed_sensor_integration_msgs/package.xml

```diff
@@ -10,12 +13,12 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>geometry_msgs</build_depend>
+  <depend>geometry_msgs</depend>
+  <depend>std_msgs</depend>
```

Comment:
Do we need std_msgs?

Reply:
As discussed it is the also in all common msg pkgs